CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
+ * version of release-bot you are using (`pip freeze | grep release-bot`)
+ * the command you executed with output
+
+If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one. You can also comment on the closed issue to indicate that upstream should provide a new release with a fix.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as GitHub issues.
+When you are creating an enhancement issue, **use a clear and descriptive title**
+and **provide a clear description of the suggested enhancement**
+in as many details as possible.
+
+## Guidelines for Developers
+
+If you would like to contribute code to the `release-bot` project, this section is for you!
+
+### Is this your first contribution?
+
+Never contributed to an open-source project before?  No problem!  We're excited that you are considering `release-bot` for your first contribution!
+
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).  It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### Dependencies
+
+If you are introducing a new dependency, please make sure it's added to:
+ * requirements.txt
+
+### Documentation
+
+We are maintaining whole project documentation inside [README.md](/README.md).
+
+#### Changelog
+
+When you are contributing to changelog, please follow these suggestions:
+
+* The changelog is meant to be read by everyone. Imagine that an average user
+  will read it and should understand the changes. `docker_image.mount() via .get_archive()` is
+  not very descriptive. `DockerImage class now utilizes get_archive() from
+  docker-py for its mount() method.` is a more friendly description.
+* Every line should be a complete sentence. Either tell what is the change that the tool is doing or describe it precisely:
+  * Bad: `Use search method in label regex`
+  * Good: `Colin now uses search method when...`
+* And finally, with the changelogs we are essentially selling our projects:
+  think about a situation that you met someone at a conference and you are
+  trying to convince the person to use the project and that the changelog
+  should help with that.
+
+
+### Testing
+
+For testing, we are using [pytest](https://docs.pytest.org/en/latest/) framework. Tests are stored in the [tests](/tests) directory. We recommend to run tests inside docker container using:
+```
+# if TEST_TARGET is empty whole test suite is executed
+make test-in-container TEST_TARGET=<PATH>
```

Comment:
Can we reword this a bit? let's do this example without TEST_TARGET and do the other one with it? And say that you can also run specific tests.